query-data/explore.mdx

```diff
@@ -181,7 +181,13 @@ To configure the table options, click <img src="/doc-assets/icons/options.svg" c
 
 Axiom can also display an event timeline about the distribution of events across the selected time range. In the event timeline, each bar represents the number of events matched within that specific time interval. Holding the pointer over a bar reveals a blue line marking the total events and shows when those events occurred in that particular time range. To display the event timeline, click <img src="/doc-assets/icons/options.svg" className="inline-icon" alt="View options icon" />, and then click **Show chart**.
 
-Drag the pointer over the event timeline to focus on the time range. The time range of your query automatically updates to match what you selected.
+#### Highlight time range on timeline
+
+Drag the pointer over the event timeline to highlight a specific time range, and then choose one of the following:
```

Comment:
only works on line charts and heat maps for now

Reply:
Thanks! Implemented